### ISSUE TITLE
Threads being skipped

### ISSUE THREAD

**Issue by irus99** [PokemonGoMap/PokemonGo-Map#774]
Is anyone else having an issue where threads are being skipped? I have 14 accounts going but it seems that only some of them are being registered.0 and 2 through 6 and 13 threads are missing

2016-08-17 15:19:59,289 [ search_worker_1][        search][    INFO] Search step 52 beginning (queue size is 39)
2016-08-17 15:19:59,823 [ search_worker_1][        models][    INFO] Upserted 0 pokemon, 20 pokestops, and 5 gyms
2016-08-17 15:20:12,168 [ search_worker_7][        search][    INFO] Search step 53 beginning (queue size is 38)
2016-08-17 15:20:13,024 [ search_worker_7][        models][    INFO] Upserted 0 pokemon, 21 pokestops, and 5 gyms
2016-08-17 15:20:14,307 [ search_worker_8][        search][    INFO] Search step 54 beginning (queue size is 37)
2016-08-17 15:20:14,749 [ search_worker_8][        models][    INFO] Upserted 0 pokemon, 22 pokestops, and 5 gyms
2016-08-17 15:20:16,458 [ search_worker_9][        search][    INFO] Search step 55 beginning (queue size is 36)
2016-08-17 15:20:16,870 [ search_worker_9][        models][    INFO] Upserted 0 pokemon, 22 pokestops, and 7 gyms
2016-08-17 15:20:18,604 [search_worker_10][        search][    INFO] Search step 56 beginning (queue size is 35)
2016-08-17 15:20:19,020 [search_worker_10][        models][    INFO] Upserted 0 pokemon, 23 pokestops, and 7 gyms
2016-08-17 15:20:20,749 [search_worker_11][        search][    INFO] Search step 57 beginning (queue size is 34)
2016-08-17 15:20:21,230 [search_worker_11][        models][    INFO] Upserted 0 pokemon, 23 pokestops, and 7 gyms
2016-08-17 15:20:22,890 [search_worker_12][        search][    INFO] Search step 58 beginning (queue size is 33)
2016-08-17 15:20:23,429 [search_worker_12][        models][    INFO] Upserted 0 pokemon, 22 pokestops, and 7 gyms
2016-08-17 15:20:29,275 [ search_worker_1][        search][    INFO] Search step 59 beginning (queue size is 32)
2016-08-17 15:20:29,809 [ search_worker_1][        models][    INFO] Upserted 0 pokemon, 23 pokestops, and 7 gyms
2016-08-17 15:20:42,164 [ search_worker_7][        search][    INFO] Search step 60 beginning (queue size is 31)
2016-08-17 15:20:42,649 [ search_worker_7][        models][    INFO] Upserted 0 pokemon, 23 pokestops, and 7 gyms
2016-08-17 15:20:44,307 [ search_worker_8][        search][    INFO] Search step 61 beginning (queue size is 30)
2016-08-17 15:20:44,889 [ search_worker_8][        models][    INFO] Upserted 0 pokemon, 23 pokestops, and 8 gyms
2016-08-17 15:20:46,459 [ search_worker_9][        search][    INFO] Search step 62 beginning (queue size is 29)
2016-08-17 15:20:47,067 [ search_worker_9][        models][    INFO] Upserted 1 pokemon, 24 pokestops, and 8 gyms
2016-08-17 15:20:48,603 [search_worker_10][        search][    INFO] Search step 63 beginning (queue size is 28)
2016-08-17 15:20:49,256 [search_worker_10][        models][    INFO] Upserted 0 pokemon, 24 pokestops, and 8 gyms
2016-08-17 15:20:50,749 [search_worker_11][        search][    INFO] Search step 64 beginning (queue size is 27)
2016-08-17 15:20:51,650 [search_worker_11][        models][    INFO] Upserted 0 pokemon, 28 pokestops, and 9 gyms
2016-08-17 15:20:52,889 [search_worker_12][        search][    INFO] Search step 65 beginning (queue size is 26)
2016-08-17 15:20:53,940 [search_worker_12][        models][    INFO] Upserted 0 pokemon, 27 pokestops, and 9 gyms
2016-08-17 15:20:59,296 [ search_worker_1][        search][    INFO] Search step 66 beginning (queue size is 25)
2016-08-17 15:21:00,234 [ search_worker_1][        models][    INFO] Upserted 0 pokemon, 53 pokestops, and 11 gyms

**Comment by justsomedudeonthenet** [comment]
They are probably banned accounts that are being stuck in a longer wait time, which is why you don't see them as often.

Run with '-d' to get the full debug log, and redirect it to a file.  Let it run for a bit then search the file for your missing workers.  They will probably be there and sleeping for a long time.

**Comment by irus99** [comment]
Thanks for your reply. The issues actually was due to my VE only have 1 core. I changed to to have 4 cores  and now I am not missing any threads on any of my accounts. Just a little more info for anyone who has the same problem. I have 3 instances running for 3 different domains. One for me and two for other people. Each domain had 14 accounts so there must be some limit per core depending on the cpu your using. I increased the vcpu count from 1 to 4 and restarted the VE and now everything working fine for all 3 instances.
